tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2895,6 +2895,8 @@ declare namespace ts {
             getAllProjectErrors(): readonly Diagnostic[];
             setProjectErrors(projectErrors: Diagnostic[]): void;
             close(): void;
+            markAsDirty(): void;
```

Comment:
How much of this are you still wanting to be public?

Reply:
This is public because for some reason Project.markAsDirty is public too ? But i can mark them internal .. not very comfortable with these methods being public at all

Reply:
Whatever you'd prefer; I don't really know whether or not anyone needs this externally. ts-eslint's use case for these is solely just "opening files" and "getting the checker"